csharp/extractor/Semmle.Extraction.CSharp/SymbolExtensions.cs

```diff
@@ -33,7 +33,7 @@ static class SymbolExtensions
         ///
         /// <param name="type">The type to disambiguate.</param>
         /// <returns></returns>
-        public static ITypeSymbol DisambiguateType(this ITypeSymbol type)
+        public static ITypeSymbol? DisambiguateType(this ITypeSymbol? type)
```

Comment:
With a slight change in implementation we could add an extra nullability attribute, which shows that the result is only null if the input was null. However now, we might get `null` back if the input was an error type symbol, but there are no type symbol candidates. Is it a hard requirement here to not return an `IErrorTypeSymbol`?